xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/Logic.js

```diff
@@ -76,7 +103,9 @@ define('xwiki-livedata', [
    */
   const Logic = function (element) {
     this.element = element;
-    this.data = JSON.parse(element.getAttribute("data-config") || "{}");
+    const data = JSON.parse(element.getAttribute("data-config") || "{}");
+    data.footnotesSet = [];
```

Comment:
Why not have this as an **internal** (private) field of ``FootnotesService``? It's only used there AFAICS.